app/components/search/results.tsx

```diff
@@ -0,0 +1,18 @@
+"use client";
+import { Suspense } from "react";
+import SearchBody from "./body";
+
+function SearchBarFallback() {
+  return <>placeholder</>;
+}
+
+const SearchResults = ({}) => {
+  return (
+    <>
+      <Suspense fallback={<SearchBarFallback />}>
```

Comment:
What was the issue with using suspense before and how did you resolve it?

Reply:
`useSearchParams()` needs to be wrapped in a `Suspense`, which means the  code that uses `useSearchParams()` needs to be in it's own component and wrapped by the `Suspense` tags. there's a config we could use to get around this but it's not recommended by Next.js and will not be supported in future versions. 

see: https://nextjs.org/docs/messages/missing-suspense-with-csr-bailout